information-schema/information-schema-variables-info.md

```diff
@@ -0,0 +1,59 @@
+---
+title: VARIABLES_INFO
+summary: 了解 information_schema 表 `VARIABLES_INFO`。
+---
+
+# VARIABLES_INFO
+
+`VARIABLES_INFO` 可用于查看当前 TiDB 集群/实例系统变量的默认值、当前值以及作用域等信息。
+
+{{< copyable "sql" >}}
```

Comment:
We no longer use this code snippet. FYI: [Docs Maintenance Changelog](https://pingcap.feishu.cn/wiki/wikcnpK4JnRsc9Zje25DLDWidLb)
```suggestion
```